modules/pel/peltool/README.md

```diff
@@ -0,0 +1,80 @@
+# peltool Usage
+
+To see all of peltool's capabilities, use the help option `peltool.py --help`.
+
+**Note:** For non-BMC environment user must pass the PEL directory to parse PELs
+          e.g. `peltool.py -p <pel_dir> -l`, `peltool.py -p <pel_dir> -n`,
+               `peltool.py -p <pel_dir> -a`
+
+## List PELs
+
+- Get list of PELs: `peltool.py -l` (By default serviceable PELs are listed)
+- Get list of PELs including non-serviceable PELs: `peltool.py -lN`
+- Get list of PELs including hidden PELs: `peltool.py -lH`
+- Get list of PELs including terminating PELs: `peltool.py -lt`
+- Get list of serviceable, non-serviceable, and hidden PELs: `peltool.py -lsNH`
+
+##### List PELs based on severity and only options
+
+- Get list of serviceable PELs and PELs with another severity: `peltool.py -l -S <Severity>`
+- Get list of PELS including PELs with severity as Informational: `peltool.py -l -S Informational`
+  User can provide multiple severities values e.g. `peltool.py -l -S Informational Critical`
+- Get list of PELs including hidden PELs only: `peltool.py -lHO`
+- Get list of PELs including critical PELs only: `peltool.py -lO -S Critical`
+
+## Show PEL Count
+
+- Get PEL count: `peltool.py -n` (By default serviceable PELs count is given)
+- Get PEL count including non-serviceable PELs: `peltool.py -nN`
+- Get PEL count including hidden PELs: `peltool.py -nH`
+- Get PEL count including terminating PELs: `peltool.py -nt`
+- Get PEL count for serviceable, non-serviceable, and hidden PELs: `peltool.py -nsNH`
+
+##### Show PEL count based on severity and only options
+
+- Get count of serviceable PELs and PELs with another severity: `peltool.py -n -S <Severity>`
+- Get PEL count including PELs with severity as Informational: `peltool.py -n -S Informational`
+  User can provide multiple severities values e.g. `peltool.py -n -S Informational Critical`
+- Get hidden PELs count only: `peltool.py -nHO`
+- Get critical PELs count only: `peltool.py -nO -S Critical`
+
+## Display all PEL data
+
+- Get all PEL data: `peltool.py -a` (By default serviceable PELs data are displayed)
+- Get all PEL data including hidden PELs: `peltool.py -aH`
+- Get all PEL data including terminating PELs: `peltool.py -at`
+- Get all PEL data for serviceable, non-serviceable, and hidden PELs: `peltool.py -asNH`
+
+##### Get all PEL data based on severity and only options
+
+- Get all PEL data based on severity value: `peltool.py -a -S <Severity>`
+- Get all data for serviceable PELs and PELs with another severity: `peltool.py -n -S <Severity>`
```

Comment:
So I think you can delete line 50, and then change the '-n' in this example to a '-a' and then I can merge.